fbw-a380x/src/systems/systems-host/systems/tcas/loop.ts

```diff
@@ -1,8 +1,8 @@
-import { TcasComponent } from '@tcas/lib/TcasComponent';
-import { TcasComputer } from '@tcas/components/TcasComputer';
+import { LegacyTcasComputer } from 'systems-host/systems/tcas/components/LegacyTcasComputer';
+import { TcasComponent } from 'systems-host/systems/tcas/lib/TcasComponent';
 
 const components: TcasComponent[] = [
-    TcasComputer.instance,
+    LegacyTcasComputer.instance,
```

Comment:
There's no need for this whole file or the awkward half-singleton/half-not nature of LegacyTcasComputer if you put TcasComputer on the backplane in systems host.